apps/notifications/src/doc/note-rendering.md

```diff
@@ -0,0 +1,105 @@
+# Notification Detail Rendering
+
+When a user clicks a note to view, that notification is rendered to the user in a slide-out drawer. The pipeline for rendering this drawer to the user begins in `/src/panel/templates/index.jsx`. When the user has selected a note to view, this component renders a child `Note` component from `src/panel/templates/note.jsx` with the property `detailView` set to `true`.
```

Comment:
What's the subtlety between a "note" and "notification"?

Reply:
They're the same thing -- both terms are used throughout the codebase and existing documentation. :/